_data/projects/WT_nnn.yml

```diff
@@ -0,0 +1,10 @@
+name: WEB_TEMPLATE_001
+desc: Cool Colurful Template for easy web development
```

Comment:
Please provide a good description here of this project and what contributions you are interested in